features/step_definitions/routability.js

```diff
@@ -126,6 +126,7 @@ module.exports = function () {
 
                 r.query = this.query;
                 r.json = JSON.parse(body);
+                r.code = r.json.code;
```

Comment:
The changes are in separate commit, but must be in previous PR:  just missed `trip` and `routability`  `code` values